app/models/uk_completion.rb

```diff
@@ -0,0 +1,13 @@
+class UkCompletion < Base
+  attribute :confirmed, :boolean, default: false
+
+  validates :confirmed, inclusion: { in: [true, false] }
+
+  def next_step
+    if confirmed == true
```

Comment:
Could use a guard here to be more concise - take it or leave it:

```
return "opt_in_emails" if confirmed
nil
```

Reply:
looks good